doc/api/errors.md

```diff
@@ -950,6 +950,18 @@ release binaries but can happen with custom builds, including distro builds.
 
 A signing `key` was not provided to the [`sign.sign()`][] method.
 
+<a id="ERR_JWK_UNSUPPORTED_CURVE"></a>
+### `ERR_JWK_UNSUPPORTED_CURVE`
```

Comment:
For consistency with the other crypto errors, should this be `ERR_CRYPTO_JWK_UNSUPPORTED_CURVE`?

Reply:
✅